src/app/deprecated/(authenticated)/user/breaches/breaches.d.ts

```diff
@@ -82,6 +82,9 @@ export interface Breach {
   Title: string;
 }
 
+/**
+ * @deprecated Use {@see SubscriberRow} instead
```

Comment:
I'm a little on the fence about this @Vinnl - I landed this PR with the comment, but I think there's at least one use case where it's required and that's when AWS can't deliver a message, we need to be able to look up by email.

Reply:
Yeah maybe the comment is a bit too stern, though I think marking it as deprecated is still fine - it won't prevent you from actually using it, but some editors will at least highlight it as something to give a second though if you can, and maybe add a comment on why it's necessary in that specific case.

(I am assuming here you meant to comment on https://github.com/mozilla/blurts-server/blob/6ed8f194c668c3bf073418bbe8bfb01d0f4b6d96/src/db/tables/subscribers.js#L84.)